File: BackEnd/Service/Models/InterviewerModel.cs
using Data.Entities;

namespace Service.Models
{
    public class InterviewerModel
    {
        public Guid InterviewerId { get; set; }

        public string UserId { get; set; }

        public virtual WebUser User { get; set; }

        public Guid CompanyId { get; set; }
        public virtual CompanyModel Company { get; set; }

        public bool IsDeleted { get; set; } = false;
    }
}

File: FRONTEND/src/app/data/interviewer/interviewer.model.ts
import { WebUser } from '../authentication/web-user.model';
import { Company } from '../company/company.model';

export class Interviewer {
	interviewerId?: string;
	userId?: string;
	companyId?: string;
	company?: Company;
	user?: WebUser;
	isDeleted?: boolean;
	daysToLastInterview?: number;
}

export class InterviewerAddModel {
	userId?: string;
	companyId?: string;
}

export class InterviewerFilterModel {
	search?: string;
	isFreeTime?: boolean;
	isBusyTime?: boolean;
	fromTime?: string;
	toTime?: string;
	fromDate?: Date;
	toDate?: Date;
}
